src/db/clients/sqlserver.js

```diff
@@ -184,6 +185,16 @@ export async function listTableTriggers(conn, table) {
   return data.map((row) => row.trigger_name);
 }
 
+export async function listTableIndexes(conn, table) {
+  // SQL Server does not have information_schema for indexes, so other way around
+  // is using sp_helpindex stored procedure to fetch indexes related to table
+  const sql = `EXEC sp_helpindex ${wrapIdentifier(table)}`;
```

Comment:
Same as the other request. Then there is no need for user template string and the `wrapIdentifier`.